insolar/message/parcel.go

```diff
@@ -31,15 +31,21 @@ type ParcelFactory interface {
 	Validate(crypto.PublicKey, insolar.Parcel) error
 }
 
-// Parcel is a message signed by senders private key.
-type Parcel struct {
-	Sender        insolar.Reference
-	Msg           insolar.Message
-	Signature     []byte
+// ServiceData is a structure with utility fields like log level and trace id.
+type ServiceData struct {
 	LogTraceID    string
+	LogLevel      insolar.LogLevel
```

Comment:
LogLevel field should be NILable. If it was not adjusted then it shouldn't affect any other node. Behaviour should be discussed in details.

Reply:
nil'able -> means NoLevel